tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Do we need all these values?

Reply:
typeRoots and types can be removed. Others are useful toggles. [Ref](https://www.typescriptlang.org/tsconfig)